be/src/http/action/mini_load.cpp

```diff
@@ -318,14 +330,47 @@ int MiniLoadAction::on_header(HttpRequest* req) {
         HttpChannel::send_basic_challenge(req, "mini_load");
         return -1;
     }
-    auto st = _on_header(req);
-    if (!st.ok()) {
-        HttpChannel::send_reply(req, HttpStatus::INTERNAL_SERVER_ERROR, st.get_error_msg());
+	
+    isStreaming = _is_streaming(req);
+    Status status;
+    if (isStreaming) {
+        status = _on_new_header(req);
+        StreamLoadContext* ctx = (StreamLoadContext*)req->handler_ctx();
+        ctx->status = status;
+    } else {
+        status = _on_header(req);
+    }
+    if (!status.ok()) {
+        HttpChannel::send_reply(req, HttpStatus::INTERNAL_SERVER_ERROR, status.get_error_msg());
         return -1;
     }
     return 0;
 }
 
+bool MiniLoadAction::_is_streaming(HttpRequest* req) {
+    // multi load must be non-streaming
+    if (!req->param(SUB_LABEL_KEY).empty()) {
+        return false;
+    }
+
+
+    const TNetworkAddress& master_address = _exec_env->master_info()->network_address;
+    TFeResult res;
+    Status status = FrontendHelper::rpc(
+            master_address.hostname, master_address.port,
+            [&res] (FrontendServiceConnection& client) {
+            client->isMiniLoadStreaming(res);
+            });
+    if (!status.ok()) {
```

Comment:
I think you should check if res is ok. status ok means this RPC call was executed successfully, but the result may be  not ok

Reply:
The res in this RPC is not important, because the result of this RPC can only be successfully when fe has been finished updating.

Reply:
this can be important now, but if we change the implementation, we can forget to change here.

And there is no harm to check res

Reply:
Em. I already changed.